src/app/share-preview/components/share-preview/share-preview.component.ts

```diff
@@ -19,22 +28,23 @@ import { EVENTS } from '@shared/services/google-analytics/events';
 import { READ_ONLY_FIELD } from '@shared/components/prompt/prompt-fields';
 import { PromptService } from '@shared/services/prompt/prompt.service';
 import { Deferred } from '@root/vendor/deferred';
+import { Dialog } from '@root/app/dialog/dialog.module';
+import { FileListComponent } from '../../../file-browser/components/file-list/file-list.component';
```

Comment:
```suggestion
import { FileListComponent } from '@fileBrowser/components/file-list/file-list.component';
```
You can import it like this as well! ...Although my IDE is saying this import isn't even needed, so actually maybe we can just delete this line.